src/arch/riscv/kernel/vspace.c

```diff
@@ -63,6 +67,28 @@ RISCVGetReadFromVMRights(vm_rights_t vm_rights)
     return vm_rights != VMWriteOnly;
 }
 
+
+pte_t pte_next(word_t phys_addr, bool_t is_leaf)
```

Comment:
If this helper is private to this file, can you declare it with `static` instead of having the function prototype?